framework/log/FileTarget.php

```diff
@@ -117,9 +120,21 @@ public function export()
             $this->rotateFiles();
             @flock($fp, LOCK_UN);
             @fclose($fp);
-            @file_put_contents($this->logFile, $text, FILE_APPEND | LOCK_EX);
+            $writeResult = @file_put_contents($this->logFile, $text, FILE_APPEND | LOCK_EX);
+            if ($writeResult === false) {
+                throw new LogRuntimeException('Unable to export log through file!');
```

Comment:
Does it make sense to use `error_get_last()` here to output more details about failure reasons?